sdk/storage/tests.yml

```diff
@@ -1,30 +1,24 @@
 trigger: none
 
 extends:
-  template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+  template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
   parameters:
     ServiceDirectory: storage
     BuildInParallel: true
     TimeoutInMinutes: 180
-    Location: canadacentral
     CloudConfig:
       Preview:
-        SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
-      PrivatePreview:
+        SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview-test)
```

Comment:
For all these `SubscriptionConfiguration` values, the `-test` suffix should be removed. That was a testing/migration keyvault secret that's very out of date now.